docs: Make minor changes to the Grouping article (#2805)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/guides/grouping.md

## Basic Usage

> To group by resources, [configure them](./resources.md) first. Then, add the [IntegratedGrouping](../reference/integrated-grouping.md) and [GroupingState](../reference/grouping-state.md) plugins.

Reviewer: It would be better. Because Integrated plugin requires State plugin

```suggestion
To group by resources, [configure them](./resources.md) first. Then, add the [GroupingState](../reference/grouping-state.md) and [IntegratedGrouping](../reference/integrated-grouping.md) plugins.
```

> To group by resources, [configure them](./resources.md) first. Then, add the [GroupingState](../reference/grouping-state.md) and [IntegratedGrouping](../reference/integrated-grouping.md) plugins.

Reviewer 2: Can you be more specific than simply "configure them"?

Author: The reader should follow the link to get all the needed information.

Reviewer 2: Okay. For the second sentence, please change it to:
Then, add [GroupingState](../reference/grouping-state.md) and [IntegratedGrouping](../reference/integrated-grouping.md) plugins.

`GroupingState` has a `grouping` property that accepts an array of resources to group by. The resource order in this array is important. For example, if the Room resource is first and Attendees second, appointments are grouped in that order. If the `grouping` property is unspecified, appointments are grouped by the [main resource](../reference/resources.md/#properties).

> Group names (resource titles) are displayed in a group panel. To show it, add the [GroupingPanel](../reference/grouping-panel.md) plugin.

Reviewer 2: In the second sentence, is "it" the group panel?
If so, please change it to:
"To show the panel..."

> A single appointment can appear in multiple groups if it is assigned to [multiple instances of a resource](./resources.md/#single-and-multiple-instance-resources). For example, an appointment assigned to three instances appears in three groups. A particular resource instance provides the appointment's color. The `members` resource from the following demo illustrates this case:

Reviewer 2: "A particular resource instance provides the appointment's color."
Change "particular" to "specific"
Also, please replace "provides" with a more concise word.

Author: ```suggestion
A single appointment can appear in multiple groups if it is assigned to [multiple instances of a resource](./resources.md/#single-and-multiple-instance-resources). For example, an appointment assigned to three instances appears in three groups. In addition, the appointment's color is taken from a specific instance. The `members` resource from the following demo illustrates this case:
```

Reviewer 2: Let's change "is taken from". Can we say:
In addition, a resource instance specifies the appointment's color.

Author: No, the color is specified inside the resource instance. The appointment uses this color. Let's not change anything then, and go with the very first version:

```suggestion
A single appointment can appear in multiple groups if it is assigned to [multiple instances of a resource](./resources.md/#single-and-multiple-instance-resources). For example, an appointment assigned to three instances appears in three groups. The appointment's color is inherited from a particular resource instance. The `members` resource from the following demo illustrates this case:
```

Reviewer 2: Okay, then please change "particular" to "specific"

> Timetable cells have the `groupingInfo` property that lists groups to which a particular cell belongs. Use this property to customize cells depending on their group.

Reviewer 2: Line 26:
Change "one _under_ another" to "one _underneath_ another"
For the third sentence, can we say:
"You can adjust group orientation based on a view, as shown in the following example:"

Line 34:
Change to: In the example below, appointments are first grouped by date and then by resource. Click the "Group by Date First" switch to change the order.

Line 46:
Change "the `groupingInfo` property" to "a `groupingInfo` property"
Change "depending on their group" to "based on their group"

> In addition, the `GroupingPanel` plugin has properties that customize the group panel: `horizontalLayoutComponent`, `rowComponent`, and `cellComponent`.

Reviewer 2: Can we improve on this by saying:
In addition, you can use the following `GroupingPanel` plugin properties to customize a group panel: `horizontalLayoutComponent`, `rowComponent`, and `cellComponent`.

> The following example shows how to use the properties mentioned above:

Reviewer 2: The following example shows how to use these properties.
